### DISCUSSION TITLE
Smalland: Survive the Wilds

### DISCUSSION THREAD

**Discussion by ssiwm** [gameserverapp/Platform#717] [Game request]
steampage: https://store.steampowered.com/app/768200/Smalland_Survive_the_Wilds/
dedicated server info: https://store.steampowered.com/news/app/768200/view/3893988945397359022?l=english
server files: https://www.dropbox.com/scl/fo/npnfynj1de24gmid2b4v6/h?rlkey=5yqehd802qy0hbsmvl59zdlme&dl=0

update
server files: https://steamdb.info/app/808040/depots/

**Comment by mvaessen** [comment]
Hopefully they upload to a proper steam repo soon. Dropbox feels a bit hacky

**Reply by ssiwm** [reply]
added ;)
